azure-pipelines.yml

```diff
@@ -11,24 +11,57 @@ schedules:
       - dev
   always: true
 
-pool:
-  vmImage: 'ubuntu-latest'
+# =================================================
 
-steps:
-- task: UsePythonVersion@0
-  inputs:
-    versionSpec: '3.5' 
-    addToPath: true
+jobs:
+- job: "Testing"
+  pool:
+    vmImage: 'ubuntu-latest'
 
-- script: pip install -r requirements.txt
-  displayName: 'Install required packages'
+  steps:
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '3.5' 
+      addToPath: true
 
-- script: flake8 .
-  displayName: "Run flake8"
+  - script: pip install -r requirements.txt
+    displayName: 'Install required packages'
 
-- script: python -m pytest test/ --junitxml=./TEST-TEST.xml 
-  displayName: 'Run tests'
+  - script: flake8 .
+    displayName: "Run flake8"
 
-- task: PublishTestResults@2
-  displayName: 'Publish Test Results **/TEST-*.xml'
-  condition: succeededOrFailed()
+  - script: python -m pytest test/ --junitxml=./TEST-TEST.xml 
+    displayName: 'Run tests'
+
+  - task: PublishTestResults@2
+    displayName: 'Publish Test Results **/TEST-*.xml'
+    condition: succeededOrFailed()
+
+# -----
+
+- job: "WindowsCodeScans"
```

Comment:
I don't see a requirement to run it for every PR anywhere.

Reply:
I think it's called trigger.

Reply:
I'd rather do that - keep anything bad out of the codebase.

It's already flagging you for having ROC everywhere :-)

Reply:
ROC is a perfectly fine acronym, and I stand by that :D 
Maybe we can add the trigger then? We can add exceptions for ROC and potential other ones

Reply:
There are definitely ways to add exceptions; just need to read up on them

Reply:
Have done so. There's now a separate pipeline

Reply:
For now, the results don't actually go anywhere, so it's not relevant.